addons/event/views/event_views.xml

```diff
@@ -587,21 +587,32 @@
                     <field name="event_ticket_id"/>
                     <templates>
                         <t t-name="kanban-box">
-                            <div class="oe_kanban_card oe_kanban_global_click">
+                            <div class="oe_kanban_card oe_kanban_global_click" style="width:400px">
                                 <div class="float-right">
                                     <a class="btn btn-sm btn-success" string="Confirm Registration" name="action_confirm" type="object" states="draft" role="button">
-                                        <i class="fa fa-check" role="img" aria-label="Confirm button" title="Confirm Registration"/>
+                                        <i class="fa fa-check fa-7x" role="img" aria-label="Confirm button" title="Confirm Registration"/>
                                     </a>
-                                    <a class="btn btn-sm btn-success" string="Attended the Event" name="action_set_done" type="object" states="open" role="button">
-                                        <i class="fa fa-level-down" role="img" aria-label="Attended button" title="Attended the Event"/>
+                                    <a class="btn btn-sm btn-success" string="Confirm Attendance" name="action_set_done" type="object" states="open" role="button">
+                                        <i class="fa fa-user-plus fa-7x" role="img" aria-label="Attended button" title="Confirm Attendance"/>
                                     </a>
+                                    <div string="Attended" type="object" states="done" role="button">
+                                        <i class="fa fa-users fa-7x" role="img" aria-label="Attended" title="Attended"/>
+                                    </div>
+                                    <div string="Cancelled" type="object" states="cancel" role="button">
+                                        <i class="fa fa-user-times fa-7x" role="img" aria-label="Cancelled" title="Cancelled"/>
+                                    </div>
                                 </div>
                                 <div class="oe_kanban_content">
                                     <div class="o_kanban_record_top">
-                                        <b class="o_kanban_record_title"><field name="display_name"/></b>
+                                        <strong class="o_kanban_record_title"><span><t t-esc="record.name.value"/></span></strong>
```

Comment:
You could just change 
`<field name="display_name" />`
to
`<field name="name" />`

Or does that not give the expected result?

Reply:
My bad, I think I did the wrong tests :)